### PR TITLE
noam - added break coast toggle switch on robot (i saw 2910 behind th…

### PR DESCRIPTION
**Pull request by greenblitz4590**
…e glass or something)

### CODE REVIEW

File: src/main/java/edu/greenblitz/tobyDetermined/Robot.java [hunk 1]
```diff
@@ -78,6 +92,7 @@ public void robotPeriodic() {
 		CommandScheduler.getInstance().run();
 		RoborioUtils.updateCurrentCycleTime();
 		SmartDashboard.putBoolean("encoderBroken", SwerveChassis.getInstance().isEncoderBroken());
+		SmartDashboard.putBoolean("switch state",BreakCoastSwitch.getInstance().isSwitchPressed());
```
Comment (NoamRosenberg08) [inline src/main/java/edu/greenblitz/tobyDetermined/Robot.java:95]: make more space and make at another chunk

File: src/main/java/edu/greenblitz/tobyDetermined/RobotMap.java [hunk 2]
```diff
@@ -20,6 +20,9 @@
 public class RobotMap {
     public static final Robot.robotName ROBOT_NAME = Robot.robotName.Frankenstein;
 
+
+    public static final int BREAK_COAST_SWITCH_DIO_PORT = 0;
```
Comment (NoamRosenberg08) [inline src/main/java/edu/greenblitz/tobyDetermined/RobotMap.java:24]: make inside of a class

File: src/main/java/edu/greenblitz/utils/BreakCoastSwitch.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
+package edu.greenblitz.utils;
```
Comment (NoamRosenberg08) [inline src/main/java/edu/greenblitz/utils/BreakCoastSwitch.java:1]: Ctrl + Alt + L the shit out of it

**Review by NoamRosenberg08 — CHANGES_REQUESTED** [review]

**Review by tnrz135 — CHANGES_REQUESTED** [review]

File: src/main/java/edu/greenblitz/tobyDetermined/RobotMap.java [hunk 4]
```diff
@@ -20,6 +20,13 @@
 public class RobotMap {
     public static final Robot.robotName ROBOT_NAME = Robot.robotName.Frankenstein;
 
+
+
+    public static class BreakCoastSwitch{
+        public static final int BREAK_COAST_SWITCH_DIO_PORT = 0;
```
[the remark this answers is not in the document]
Comment (NoamRosenberg08) [reply src/main/java/edu/greenblitz/tobyDetermined/RobotMap.java:26]: the port is acctually 0

**Pull request merged**